model_params:
  weights_path: "./traversability_model_trace_distill128_cfs.pt"
  static_output_dim: 32
map_params:
  resolution: 0.1
  height: 25.6
  width: 25.6
  origin_x: 12.8
  origin_y: 12.8
  # map_to_world: [0.0, 5.0, -2.0]
  map_to_world: [0.0, 0.0, 0.0]
planner_params:
  max_v: 1.0
  # max_w: 1.57079
  max_w: 0.3
  # max_w: 0.0 
  max_dv: 0.2
  # max_dw: 0.78539
  max_dw: 0.15
  partitions: 11
  dt: 1.0
  max_iters: 10000
  # Set max time to 0.0 to not use 
  max_time: 0.0
  goal_tolerance: 0.5
  cost_weight: 1.5
input_topics:
  image: "/stereo/left/image_raw/compressed"
  pointcloud: "/ouster/points"
  camera_info: "/stereo/left/camera_info"
output_topics:
  depth: "/stereo/left/image_raw/depth"
  traversability: "/creste/bev/traversability"
  semantic_elevation: "/creste/3d/semantic_elevation"
viz_params:
  history_window: 3
  viz_3d: False

pt2pix: # LiDAR to Rectified Camera Pixels
  rows: 3
  cols: 4
  data: [
    305.22870, -371.22992, -32.38245, 87.04788,
    208.37674, 4.84756, -397.81979, 70.21586,
    0.99492, 0.01250, -0.09979, 0.09980
  ]
pix2pt: # Rectified Camera Pixels to LiDAR
  rows: 4
  cols: 4
  data: [
    0.000128,     -0.001065,      1.051069,     -0.088983,
    -0.010665,      0.000050,      0.815308,      0.149835,
    -0.000063,     -0.010612,      0.560481,      0.131719,
    -0.000000,      0.000000,      0.000000,      1.000000
  ]
